src/dat/gui/GUI.js

```diff
@@ -1131,23 +1131,30 @@ function recallSavedValue(gui, controller) {
 }
 
 function add(gui, object, property, params) {
-  if (object[property] === undefined) {
-    throw new Error(`Object "${object}" has no property "${property}"`);
-  }
-
   let controller;
+  
+  if (object instanceof Controller) {
+    controller = object;
+    params = property || { };
```

Comment:
`params` doesn't take a default value normally, so is it OK to omit the `|| {}` here?